conans/test/functional/toolchains/cmake/cmakedeps/test_link_order.py

```diff
@@ -226,7 +226,9 @@ def _get_link_order_from_cmake(content):
         if 'main.cpp.o -o example' in line:
             _, links = line.split("main.cpp.o -o example")
             for it_lib in links.split():
-                if it_lib.startswith("-l"):
+                if it_lib.startswith("-L") or it_lib.startswith("-Wl,"):
```

Comment:
Why ``-Wl,`` needs to be skipped here too? Seems a bit unexpected, I see why filtering -L, but not this one